mobile/src/main/res/drawable-v26/ic_qu_camera_launcher.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@color/ic_shortcuts_background"/>
+    <foreground>
+        <inset
+            android:drawable="@drawable/ic_qu_camera_foreground"
+            android:inset="9.26%"/>
```

Comment:
What's with this magic number?

Reply:
![1_6klde9oxebwfwom7plvcqa 1](https://user-images.githubusercontent.com/5838634/38123330-1cc38af6-340d-11e8-81a4-e308da0007fb.png)
AdaptiveIconDrawable will scale the supplied asset to 108dp, so to calculate the inset required to produce a 44dp icon: 48 / 24 * 44 = 88; that is we need to inset the scaled up asset by 10dp each side: 10 / 108 → 9.26%
https://medium.com/google-developers/implementing-adaptive-icons-1e4d1795470e

Reply:
> To use the same 48dp file I needed to inset it so that the icon is the correct size once it’s scaled up (yay vectors!) to the 108dp adaptive icon size. The background is achieved using a ColorDrawable:

But here you're using 108dp icons?

Reply:
The actual effect tells me that there is no problem